### DISCUSSION TITLE
How do I implement a 'since' version tag with hyperlink to the release note?

### DISCUSSION THREAD

**Discussion by GarfieldJiang** [squidfunk/mkdocs-material#7321] [Q&A] [answered]
Like in this document https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/
![image](https://github.com/squidfunk/mkdocs-material/assets/1414571/a02f788d-069e-44aa-a1ad-dd5176c1129b)
in the red block.

**Comment by alexvoss** [comment, marked as answer]
Please always check the issues and discussions boards before posting - this is a [frequently asked question](https://github.com/squidfunk/mkdocs-material/discussions?discussions_q=shortcodes). What you are looking at are shortcodes, which are a customization used in our own documentation as part of the conventions used.

**Reply by GarfieldJiang** [reply]
Sorry didn't do it.

**Reply by alexvoss** [reply]
No worries. I was not complaining - just a good idea generally to have a look first.

**Reply by GarfieldJiang** [reply]
Yup, bad thing is that I didn't know the existence of the word 'shortcode'. :P

**Reply by alexvoss** [reply]
Just checked and 'convention' would have found the same info. I will see if we can include something on this in the documentation itself. As I mentioned, this is a frequently asked question, so you are not the only one who is interested in the shortcodes and also not the only one who does not find the information.
